README.md

```diff
@@ -2,28 +2,53 @@
 
 This is a collection of useful tools and resources for managing an osu! tournament.
 
-<details open markdown="block">
-  <summary>
-    Table of contents
-  </summary>
-  {: .text-delta }
-1. TOC
-{:toc}
+<details open markdown="block"> 
+  <summary>Table of Contents</summary>
```

Comment:
The ToC is already functional if you check https://awesome-osu.subject.moe/

The `{:toc}` generates it based on the headers (h1 is top level, h2 is a sub header of h1, etc), so this is not needed. 

If you wish to customise it differently you could check out: https://github.com/toshimaru/jekyll-toc